benchmark/engine.py

```diff
@@ -0,0 +1,324 @@
+import json
+import os
+import sys
+import time
+import uuid
+from enum import Enum
+
+import boto3
+import dialogflow_v2 as dialogflow
+import soundfile
+
+
+def _path(x):
+    return os.path.join(os.path.dirname(__file__), '../%s' % x)
+
+sys.path.append(_path('rhino/binding/python'))
+from rhino import Rhino
+
+
+class NLUEngines(Enum):
+    AMAZON_LEX = 'AMAZON_LEX'

```

Comment:
let's do only Lex or change Dialogflow to GoogleDialogflow.

Reply:
I'll change Dialogflow to GoogleDialogflow